src/components/prose.tsx

```
@@ -0,0 +1,20 @@
+  html: string;
+}
+
+export function Prose({ children, html, className }: ProseProps) {
+  return (
+    <article
+      className={cn(
+        "mx-auto prose prose-sm md:prose-base max-w-none dark:prose-invert prose-h1:text-xl prose-h2:font-semibold prose-p:text-justify prose-a:text-blue-600",
+        className,
+      )}
+    >
+      {html ? <div dangerouslySetInnerHTML={{ __html: html }} /> : children}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)